tests/Framework/MockPlugin.php

```diff
@@ -50,6 +54,10 @@ public function addResponse($response)
                 throw new InvalidArgumentException('Invalid response');
             }
             $response = self::getMockFile($response);
+
+            if ($response === false) {
+                throw new InvalidArgumentException('Unable to create a response object from the file path');
+            }
```

Comment:
We weren't checking if the response object was being successfully created from the file path before.